core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -242,34 +244,20 @@ func (c *coordinator) ReportBlocks(
 		return
 	}
 
-	beaconLogs, err := c.lp.LogsWithSigs(
-		currentHeight-c.lookbackBlocks,
-		currentHeight,
-		[]common.Hash{
-			c.newTransmissionTopic,
-		},
-		c.beaconAddress,
-		pg.WithParentCtx(ctx))
-	if err != nil {
-		err = errors.Wrapf(err, "logs with topics. address: %s", c.beaconAddress)
-		return
-	}
-
-	logs = append(logs, beaconLogs...)
 	c.lggr.Trace(fmt.Sprintf("vrf LogsWithSigs: %+v", logs))
 
 	randomnessRequestedLogs,
 		randomnessFulfillmentRequestedLogs,
 		randomWordsFulfilledLogs,
-		newTransmissionLogs,
+		outputsServedLogs,
 		err := c.unmarshalLogs(logs)
 	if err != nil {
 		err = errors.Wrap(err, "unmarshal logs")
 		return
 	}
 
-	c.lggr.Trace(fmt.Sprintf("finished unmarshalLogs: RandomnessRequested: %+v , RandomnessFulfillmentRequested: %+v , RandomWordsFulfilled: %+v , NewTransmission: %+v",
-		randomnessRequestedLogs, randomWordsFulfilledLogs, newTransmissionLogs, randomnessFulfillmentRequestedLogs))
+	c.lggr.Trace(fmt.Sprintf("finished unmarshalLogs: RandomnessRequested: %+v , RandomnessFulfillmentRequested: %+v , RandomWordsFulfilled: %+v , OutputsServed: %+v",
+		randomnessRequestedLogs, randomWordsFulfilledLogs, outputsServedLogs, randomnessFulfillmentRequestedLogs))
```

Comment:
The order of the args in the `Sprintf` is incorrect - should be :
```suggestion
	c.lggr.Trace(fmt.Sprintf("finished unmarshalLogs: RandomnessRequested: %+v , RandomnessFulfillmentRequested: %+v , RandomWordsFulfilled: %+v , OutputsServed: %+v",
		randomnessRequestedLogs, randomnessFulfillmentRequestedLogs, randomWordsFulfilledLogs, outputsServedLogs))
```